runner/src/artifacts.rs

```diff
@@ -388,3 +458,71 @@ pub fn list_artifacts(args: &ArtifactArgs) -> ExitCode {
 
     ExitCode::SUCCESS
 }
+
+// ————————————————————————————— Process disk image ————————————————————————————— //
+
+pub fn download_disk_image(name: &str, url_path: &str) -> PathBuf {
+    let file_name: &str = extract_file_name(url_path);
+
+    if !is_file_present(&format!("artifacts/{}-miralis.img", name)) {
+        // Download image
+        log::info!("Disk image not found. Fetching the image...");
+        let mut image_downloader = Command::new("wget");
```

Comment:
I am not sure if `wget` is available everywhere (e.g. on Windows), but good enough for now :)
As a general rule, `curl` is the one that is almost always installed, including on Windows.